packages/zcli-themes/src/commands/themes/update.ts

```diff
@@ -0,0 +1,46 @@
+import { Command, Flags, CliUx } from '@oclif/core'
+import * as path from 'path'
+import * as chalk from 'chalk'
+import createThemeUpdateJob from '../../lib/createThemeUpdateJob'
+import createThemePackage from '../../lib/createThemePackage'
+import uploadThemePackage from '../../lib/uploadThemePackage'
+import pollJobStatus from '../../lib/pollJobStatus'
+
+export default class Update extends Command {
+  static description = 'import a theme'
+
+  static flags = {
+    themeId: Flags.string({ description: 'The id of the theme to update' }),
+    replaceSettings: Flags.boolean({ default: false, description: 'Whether or not to replace the current theme settings' })
+  }
+
+  static args = [
+    { name: 'themeDirectory', required: true, default: '.' }
+  ]
+
+  static examples = [
+    '$ zcli themes:import ./copenhagen_theme'
+  ]
+
+  static strict = false
+
+  async run () {
+    let { flags: { themeId, replaceSettings }, argv: [themeDirectory] } = await this.parse(Update)
+    const themePath = path.resolve(themeDirectory)
+
+    themeId = themeId || await CliUx.ux.prompt('Theme ID')
+
+    const job = await createThemeUpdateJob(themeId, replaceSettings)
+    const { readStream, removePackage } = await createThemePackage(themePath)
+
+    try {
+      await uploadThemePackage(job, readStream)
+    } finally {
+      removePackage()
+    }
+
+    await pollJobStatus(themePath, job.id)
+
+    this.log(chalk.green('Theme updated successfully'))
```

Comment:
```suggestion
 this.log(chalk.green('Theme imported successfully'), `theme ID: ${job.data.theme_id}`)
```

For consistency with [import](https://github.com/zendesk/zcli/blob/master/packages/zcli-themes/src/commands/themes/import.ts#L45) and [publish](https://github.com/zendesk/zcli/pull/181/files#diff-c5c2ff46ec004beb410f94a963da84cf7eb3a729d40e780fc56020daa82398eeR37)?

Reply:
Sorry I forgot to mark my comment as a nit pick 🙈  It was more because we also output the theme ID for [publish](https://github.com/zendesk/zcli/pull/181/files#diff-c5c2ff46ec004beb410f94a963da84cf7eb3a729d40e780fc56020daa82398eeR37) even though the id is an argument. It's a nit pick though.